community/people/aps/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: person
+title: Andrew Schumaker
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+phone: "218/4608408"
+city_state_zip: "Fitchburg, WI 53711"
+services:
+  github: inquisitor-schumaker
+---
+
+Andrew Schumaker (Shoe-maker) is a nuclear engineering graduate student at the University of Wisconsin-Madison. His undergraduate degree is in physics from the United States Military Academy, where he graduated in 2011. He will be researching nonproliferation. 
```

Comment:
1. Wisconsin —  Madison should use an em dash with spaces around the em dash.
2. Change "will be" to present tense.